src/main/java/training_day3/HW/Zad1HW.java

```diff
@@ -0,0 +1,17 @@
+package training_day3.HW;
+
+public class Zad1HW {
+
+    public static void main(String[] args){
+
+    }
+
+    private static int Smallest(int a, int b, int c){
+        if(a < b)
+            return a < c? a:c;
```

Comment:
The ternary operator ca be formatted as so: a < c ? a : c.

Reply:
tova za spac-ovete li samo stava duma?

Reply:
Please, write on english. And yes.